README.md

```diff
@@ -1,64 +1,99 @@
 # Sample Project
 
-This is as an example on how teams can structure their project repositories. Thanks to Lindsey Heagey and Joachim Meyer for the template!
+This is an example of how teams can structure their project repositories and format their project README.md file.
 
-## Files
+When creating a project repository from this template, we recommend choosing "Public" (so other participants can follow progress). You can also select Private initially and make Public before final presentation. We also recommend adding a "topic" to your repository details (click on the gear icon next to the "About" section on the repository page) to help others find your work (e.g. `uw-hackweek-snowex2024`).
```

Comment:
Do you think "You can also select Private initially and make Public before final presentation" is still fair, or also remove it? I have seen the instances where participants are shy about committing any unfinished or unpolished work because it is public, but that is something I hope we can help overcome during the hackweek :)

Reply:
Thanks! I've updated this to only mention creating a public repo.